R/teal_slices-store.R

```diff
@@ -17,6 +17,21 @@
 #'
 #' @return `NULL`, invisibly.
 #'
+#' @examples
+#' # use non-exported function from teal
+#' slices_store <- getFromNamespace("slices_store", "teal")
+#'
+#' # Create a teal_slices object
+#' tss <- teal_slices(
+#'   teal.slice::teal_slice(dataname = "data", varname = "var"),
+#'   teal.slice::teal_slice(dataname = "data", expr = "x > 0", id = "positive_x", title = "Positive x")
+#' )
+#'
+#' if (interactive()) {
+#'   # Store the teal_slices object to a file
+#'   slices_store(tss, "path/to/file.json")
+#' }
+#'
```

Comment:
How about we merge these 2 man pages (`slices_store` and `slices_restore`) and take advantage of `withr::local_file` to generate files that the developer can use/inspect

Alternatively, add a `slices_store` example to the `slices_restore` man page so that the example works